app/__init__.py

```diff
@@ -4,4 +4,8 @@
 def create_app(test_config=None):
     app = Flask(__name__)
 
+    from .planet_routes import bp
+    app.register_blueprint(bp)
```

Comment:
👍 

If you move the `planet_routes.py` file this will need to change a little.